fe/fe-core/src/main/java/com/starrocks/catalog/TableFunctionTable.java

```diff
@@ -190,7 +205,7 @@ private void parseProperties() throws DdlException {
             throw new DdlException("format is null. Please add properties(format='xxx') when create table");
         }
 
-        if (!format.equalsIgnoreCase("parquet") && !format.equalsIgnoreCase("orc")) {
+        if (!SUPPORTED_FORMATS.contains(format.toLowerCase())) {
```

Comment:
Why not use getFileFormatType() first to convert `format` to TFileFormatType (as the standard internal format description), then check whether it's supported?

Reply:
The `getFileFormatType` is used by broker load and `files`, but this check is only available in `files`.